src/ldp/http/BasicResponseWriter.ts

```diff
@@ -12,7 +12,7 @@ export class BasicResponseWriter extends ResponseWriter {
   public async canHandle(input: { response: HttpResponse; result: ResponseDescription | Error }): Promise<void> {
     if (!(input.result instanceof Error)) {
       if (input.result.body && !input.result.body.binary) {
```

Comment:
Why double `if` here? (and why doesn't the linter guard this?)

Reply:
True, actually surprised it didn't complain about this.